### ISSUE TITLE
[Suggestion] - Gravity Rework

### ISSUE THREAD

**Issue by zalexville** [tsunamayo/Starship-EVO#2160]
### Short Description of the issue:

I am enjoying working on a large ship in SEVO. While working, I have noticed two potential quality-of-life problems. My design, for example, is compact. I have no areas outside of the clean halls to fit gravity. While yet I could potentially build it into the design, it would complete remove what I am aiming to create. 

Another thing is the default field. Maybe instead of having a specificed field for the gravity, switch it around. A single block can produce gravity at the direction the player wants. The field would automatically take the entire ship and apply gravity. A seperate block can produce anti-gavity and could work similarly to how the current gravity blocks work, just the other way around.

**Comment by tsunamayo** [comment]
Yes I wanted to add that, a single block global gravity. But that block would have to be scaled up for large ship, ie 2x2x2 for say 75m, 3x3x3 for 150 and so on.
Thanks
